schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -152,39 +152,6 @@
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers_immutabilityPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers_immutabilityPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers_immutabilityPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_fileServices" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_fileServices_shares" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2019-06-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_privateEndpointConnections" },
```

Comment:
Why remove them? Will we add it to somewhere else?

Reply:
you're right, new refs are added in [autogeneratedResources.json](https://github.com/Azure/azure-resource-manager-schemas/pull/973/files/91e7681c2c68648204ea20ba9a6270fb3a7c906a#diff-d4739cbf75decf81aeb58113dcc750afR3877-R3993) , and it is referenced back in this file here: https://github.com/Azure/azure-resource-manager-schemas/blob/91e7681c2c68648204ea20ba9a6270fb3a7c906a/schemas/2019-04-01/deploymentTemplate.json#L1260-L1262

Reply:
Good for me.